code/frameworks/angular/package.json

```
@@ -105,7 +105,7 @@
     "@nrwl/workspace": "14.6.1",
     "rxjs": "^6.0.0 || ^7.4.0",
     "typescript": "^4.0.0",
-    "zone.js": "^0.8.29 || ^0.9.0 || ^0.10.0 || ^0.11.0"
+    "zone.js": "^0.8.29 || >= 0.9.0"
```

@yannbf Related to our discussion in slack, do we really want to continue to add new minor versions here? Or should we follow semver versioning instead and assume only a breaking change with a major version bump? So I could think of a solution like this as well:

```suggestion
    "zone.js": "^0.8.29 || >= 0.9.0 < 1.0.0"
```